### PR TITLE
Fix add related coding standard violations

### LINKED ISSUE #259: Add command related coding violations

### PR DESCRIPTION
**Pull request by OVReader**
Fix #259

### CODE REVIEW

File: docs/diagrams/PlantUML File/CommandAddParserRelatedClassesDiagram.puml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+abstract class "{abstract}\nCommandAddParser" as CommandAddParser
+class CommandAddPropertyParser
+class CommandAddClientParser
 
-ParseAdd <|-- ParseAddProperty
-ParseAdd <|-- ParseAddClient
+CommandAddParser <|-- CommandAddPropertyParser
+CommandAddParser <|-- CommandAddClientParser
 
 skinparam classAttributeIconSize 0
```
Comment (ngdeqi) [inline docs/diagrams/PlantUML File/CommandAddParserRelatedClassesDiagram.puml:11]: Good job changing PlantUML settings to follow the convention taught in class

File: src/main/java/seedu/duke/ClientList.java [hunk 2]
```diff
@@ -6,12 +8,13 @@
...
 public class ClientList {
-    private static int currentListSize;
     private static ArrayList<Client> clientList;
+    private static int currentListSize;
+    private static final int INITIAL_LIST_SIZE = 0;
 
     public ClientList() {
         clientList = new ArrayList<>();
-        currentListSize = 0;
+        currentListSize = INITIAL_LIST_SIZE;
```
Comment (ngdeqi) [inline src/main/java/seedu/duke/ClientList.java:17]: good job converting this to a constant

File: src/test/java/seedu/duke/parsermanager/CommandAddClientParserTest.java [hunk 3]
```diff
@@ -77,75 +78,77 @@ public void parseCommand_validAddClientInput_matchingClientDetails() throws Duke
     @Test
     public void checkForEmptyAddClientDetails_emptyClientDetail_exceptionThrown() {
         ClientList clientList = new ClientList();
-        ParseAddClient parseAddClient = new ParseAddClient(EMPTY_CLIENT_INPUT, clientList);
-        assertThrows(EmptyAddClientDetailException.class, parseAddClient::parseCommand);
+        CommandAddClientParser commandAddClientParser = new CommandAddClientParser(EMPTY_CLIENT_INPUT, clientList);
+        assertThrows(EmptyAddClientDetailException.class, commandAddClientParser::parseCommand);
```
Comment (ngdeqi) [inline src/test/java/seedu/duke/parsermanager/CommandAddClientParserTest.java:82]: Interesting use of ::

**Review by ngdeqi — COMMENTED** [review]
Follows code quality standards. LGTM

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -177,7 +177,7 @@ Also, most of the sub-methods are used to perform validations on the extracted d
 - Common:
     - `checkForEmptyDetails(String)`: Checks for any missing essential details, non-essential detail such as optional email can be empty.
 
-Note: Since the target user is a property manager working in Singapore, some validations are tailored to Singapore context.
+**Note**: Since the target user is a property manager working in Singapore, some validations are tailored to Singapore context.
```
Comment (wilsonngja) [inline docs/DeveloperGuide.md:180]: This is a good place to indicate a bold to draw the reader attention. Good job.

File: src/main/java/seedu/duke/Client.java [hunk 5]
```diff
@@ -4,11 +4,12 @@
...
-    public static final String CLIENT_LABEL = "  Client: ";
-    public static final String CONTACT_NUMBER_LABEL = "  Contact Number: ";
-    public static final String EMAIL_LABEL = "  Email: ";
-    public static final String BUDGET_SGD_LABEL = "  Budget: SGD";
-    public static final String PER_MONTH_LABEL = "/month";
+    private static final String CLIENT_LABEL = "  Client: ";
+    private static final String CONTACT_NUMBER_LABEL = "  Contact Number: ";
+    private static final String EMAIL_LABEL = "  Email: ";
+    private static final String BUDGET_SGD_LABEL = "  Budget: SGD";
+    private static final String PER_MONTH_LABEL = "/month";
```
Comment (wilsonngja) [inline src/main/java/seedu/duke/Client.java:11]: Good use of constant string :)

File: src/main/java/seedu/duke/Storage.java [hunk 6]
```diff
@@ -9,8 +9,8 @@
 import seedu.duke.exception.storageexception.PairFileNotFoundException;
 import seedu.duke.exception.storageexception.PropertyFileNotFoundException;
 import seedu.duke.exception.storageexception.StorageException;
-import seedu.duke.parsermanager.ParseAddClient;
-import seedu.duke.parsermanager.ParseAddProperty;
+import seedu.duke.parsermanager.add.CommandAddClientParser;
+import seedu.duke.parsermanager.add.CommandAddPropertyParser;
```
Comment (wilsonngja) [inline src/main/java/seedu/duke/Storage.java:13]: Good job in putting them into a package.

File: src/main/java/seedu/duke/command/add/CommandAddProperty.java [hunk 7]
```diff
@@ -22,7 +22,14 @@ public class CommandAddProperty extends CommandAdd {
     private final String rentingPrice;
     private final String unitType;
 
-    private static final Logger logger = Logger.getLogger("CommandAddProperty");
+    private static final int LANDLORD_INDEX = 0;
+    private static final int PROPERTY_ADDRESS_INDEX = 1;
+    private static final int PROPERTY_PRICE_INDEX = 2;
+    private static final int PROPERTY_UNIT_TYPE_INDEX = 3;
```
Comment (wilsonngja) [inline src/main/java/seedu/duke/command/add/CommandAddProperty.java:28]: I really like how u specifically made the effort to prevent magic literals

**Review by wilsonngja — APPROVED** [review]
Good job on fixing the standard violation :D LGTM

**Pull request merged**